app/views/search/index.html.erb

```diff
@@ -4,7 +4,7 @@
   <div class="results">
       <% @gifs.each do |gif| %>
         <div class="gif">
-        <img src= "<%= gif[:images][:original][:url]%>"> <br/>
+          <img src= "<%= gif.image%>">
```

Comment:
Did this end up working with the image_tag too? Just curious, either way is fine.

Reply:
I need to try that. It should!